test/service/test_response.py

```diff
@@ -2654,76 +2680,110 @@ def tearDownClass(cls):
         cls._teardown_indices()
         super().tearDownClass()
 
-    def _count_total_cells(self, response_json):
-        """
-        Return the number of cell suspension inner entities and total cell count
-        per hit.
-        """
-        return [
-            (
-                len(hit['cellSuspensions']),
-                sum([cs['totalCells'] for cs in hit['cellSuspensions']])
-            )
-            for hit in response_json['hits']
-        ]
-
     def test_sorting_by_cell_count(self):
         """
-        Verify sorting by 'cellCount' sorts the documents based on the total
-        number of cells in each document, using the sum of total cells when a
-        document contains more than one cell suspension inner entity.
+        Verify sorting projects by the total cell suspension cell count, the
+        project cell count, and the effective cell count.
         """
-        ascending_results = [
-            (1, 1),
-            (1, 349),
-            (1, 6210),
-            (2, 7738),
-            (1, 10000)
-        ]
-        for ascending in (True, False):
-            with self.subTest(ascending=ascending):
-                params = {
-                    'catalog': self.catalog,
-                    'sort': 'cellCount',
-                    'order': 'asc' if ascending else 'desc'
-                }
-                url = self.base_url.set(path='/index/projects', args=params)
-                response = requests.get(str(url))
-                response.raise_for_status()
-                response_json = response.json()
-                actual_results = self._count_total_cells(response_json)
-                expected = ascending_results if ascending else list(reversed(ascending_results))
-                self.assertEqual(actual_results, expected)
+        test_cases = {
```

Comment:
Did you have to adjust the expectations to accommodate the change in field type?

Reply:
The change in field type (`pass_thru_int` to `null_int`) did not affect the results in the test of sorting, however the change was required in order to filter by a cell count of `None`.

Reply:
Makes sense.